### PR TITLE
chore(RHIF-220): Update /beta to /preview

### PR DESCRIPTION
**Pull request by johnsonm325**
To test, you can run via `/beta` or `/preview`. We previously had issues with the local app being served. To ensure that it is properly being served, run the app via beta and make a change in the code or add a console log and ensure that you're getting local changes.

### CODE REVIEW

**Review by Fewwy — APPROVED** [review]
Tested it locally. Everything works as expected LGTM

**Comment by gkarat** [comment]
Other than that LGTM :) let's make sure we are not breaking anything with `basename` change, and then we are ready to merge it

File: profiles/local-frontend-and-api.js
[the remark this answers is not in the document]
Comment (johnsonm325) [reply profiles/local-frontend-and-api.js:1]: I don't think it has a purpose anymore, but I wasn't confident in deleting it. :)

**Comment by johnsonm325** [comment]
@gkarat wanna take another peek?

**Comment by Fewwy** [comment]
I'm not getting local changes. I added console.log to the RecsListTable.js and ran the app using npm run start:beta as always.

**Review by gkarat — APPROVED** [review]
LGTM! Thanks Michael

**Pull request merged**